modules/installation-vsphere-machines.adoc

```diff
@@ -85,7 +78,7 @@ $ base64 -w0 <installation_directory>/master.ign > <installation_directory>/mast
 +
 [source,terminal]
 ----
-$ base64 -w0 <installation_directory>/worker.ign > <installation_directory>/worker.64
+$ base64 -w0 <installation_directory>/compute.ign > <installation_directory>/compute.64
```

Comment:
Before this PR, these commands used `worker` in the filename. Updated to `compute` to align with wording in rest of this module.